api/routers/prediction.py

```diff
@@ -55,8 +55,12 @@ class Prediction(BaseModel):
 @router.get("/v1/prediction", tags=["Prediction"])
 async def get_account_prediction_result(name: str, breakdown: Optional[bool] = False):
     """
-    Selects a player's prediction from the plugin database.\n
-    Use: Used to determine the prediction of a player according to the prediction found in the prediction table.
+    Parameters:
+        name: The name of the player to get the prediction for
+        breakdown: If True, return the breakdown of the prediction
```

Comment:
"If True, always return the breakdown of the prediction. Otherwise, breakdown will be omitted in special cases."
Is this better?

Reply:
fixed